sql/core/src/test/resources/sql-tests/results/pgSQL/boolean.sql.out

```diff
@@ -39,7 +39,7 @@ SELECT boolean('   f           ') AS `false`
 -- !query 4 schema
 struct<false:boolean>
 -- !query 4 output
-NULL
+false
```

Comment:
plz remove the comments, too: https://github.com/apache/spark/blob/d502c80404c398d852dfa5f86a0e87c104a6286e/sql/core/src/test/resources/sql-tests/inputs/pgSQL/boolean.sql#L25

Reply:
I found the four places to remove.

Reply:
done Thank you @maropu